spec/requests/providers/means_reports_spec.rb

```diff
@@ -10,7 +10,12 @@
   let(:before_subject) { nil }
 
   describe 'GET /providers/applications/:legal_aid_application_id/means_report' do
-    subject { get providers_legal_aid_application_means_report_path(legal_aid_application, debug: true) }
+    subject do
+      # dont' match on path - webpacker keeps changing the second part of the path
```

Comment:
Very minor nitpick -  this shoud be`don't` not `dont'`. Occurs on all four tests. Not a blocker though.